tests/unit/mesh/parallel/test_shiftedmetric.cxx

```diff
@@ -38,28 +43,28 @@ class ShiftedMetricTest : public ::testing::Test {
     Field3D input_temp{mesh};
 
     fillField(input_temp, {{{1., 2., 3., 4., 5.},
+                            {2., 1., 3., 4., 5.},
+                            {1., 3., 2., 4., 5.},
+                            {1., 2., 4., 3., 5.},
+                            {1., 2., 3., 5., 4.},
```

Comment:
Why have these changed? Was there a (potential) bug the previous pattern was hiding? This might need a comment

Reply:
There was a bug, fixed in c2f9eff, that the previous pattern missed because the input was constant in y. Will add a comment.